AWS/GoCardlessConfig.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace ToucanTech\AWS;

```

Comment:
:notebook: Suggestion, you could improve the structure and relocate this config into AWS/Payments/ this would allow cleaner way for users to find all the payment gateway files without them getting lost in the AWS root.